packages/ra-ui-materialui/src/list/FilterFormInput.js

```diff
@@ -3,23 +3,38 @@ import PropTypes from 'prop-types';
 import { Field } from 'redux-form';
 import IconButton from '@material-ui/core/IconButton';
 import ActionHide from '@material-ui/icons/HighlightOff';
+import { makeStyles } from '@material-ui/core/styles';
 import classnames from 'classnames';
 import { useTranslate } from 'ra-core';
 
 const emptyRecord = {};
 
 const sanitizeRestProps = ({ alwaysOn, ...props }) => props;
 
-const FilterFormInput = ({ filterElement, handleHide, classes, resource }) => {
+const useStyles = makeStyles(theme => ({
+    body: { display: 'flex', alignItems: 'flex-end' },
+    spacer: { width: '1em' },
```

Comment:
It would be better to use `theme.spacing` here too so that spaces are consistent